gson/src/main/java/com/google/gson/GsonBuilder.java

```diff
@@ -325,6 +327,32 @@ public GsonBuilder setFieldNamingStrategy(FieldNamingStrategy fieldNamingStrateg
     return this;
   }
 
+  /**
+   * Configures Gson to apply a specific number strategy during deserialization for {@link Object}.
+   *
+   * @param objectToNumberStrategy the actual object-to-number strategy
+   * @return a reference to this {@code GsonBuilder} object to fulfill the "Builder" pattern
+   *
+   * @see ToNumberPolicy#DOUBLE
```

Comment:
Maybe make it more clear here that this is the default (same for `setNumberToNumberStrategy`)? E.g. by mentioning it in the documentation and not adding a `@see`